integration/test/bootstrap/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd project/
+go install .
+
+# Start the bootstrap process.
+apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
```

Comment:
This script configured via the integration test job installs the CLI and starts the bootstrap.

Reply:
👍